Jenkinsfile

```diff
@@ -82,4 +82,5 @@ void executeUnitTestsWithCoverage() {
     sh label: 'Running unit tests', script: 'pipenv run unittests-cov'
     sh label: 'Displaying code coverage report', script: 'pipenv run coverage-report'
     sh label: 'Exporting code coverage report', script: 'pipenv run coverage-report-xml'
+    sh label: 'Running SonarQube analysis', script: "sonar-scanner -D${SONAR_HOST} -D${SONAR_TOKEN}""
```

Comment:
Can you add these two variables to the [pipeline README's Permissions and global variables section](https://github.com/nhsconnect/integration-adaptors/blob/RT-23-static-analysis/pipeline/README.md#permissions-and-global-variables) please?